NewForce-at-Mountwest/bangazon-workforce-bucatini
# Ticket #8

Users are able to view all, create, and delete computers (if computer has not been assigned to user)

To test:
Click "Computer List" tab, user should see all computers listed.
Click on "Add New" link, user should be directed to view where computer manufacturer, make, and purchase date can be entered. Clicking on "Submit" will send instance of new computer to database.
From "Index" of computers, user should be able to click "delete" of one computer, and be sent to a form with that computer's information displayed with a confirmation of delete presented. If the user clicks delete, and the computer has ever been assigned an employee, the user will be redirected to index and the computer will not be deleted. If the computer has never had an assigned employee, it will be deleted from the DB.

[ N ] I have not commented the code in tough to understand errors
[ ? ] My code produces no new errors
[ Y ] I haz the head aches
